src/server/Html.js

```diff
@@ -34,6 +35,18 @@ export default function Html({
         <meta property="description" content="Team transparency, made easy." />
         <style dangerouslySetInnerHTML={{__html: lookCSSToken}} id={lookConfig.styleElementId} />
         <title>{title}</title>
+        {/* segment.io analytics */}
+        {process.env.SEGMENT_WRITE_KEY &&
```

Comment:
I got what you meant, I just didn't want to have to go and create _another_ build task. The thing is, you're 100% right. I'll do the right thing 💪